cypress/TestCases/README.md

```diff
@@ -1,33 +1,20 @@
-# Firebolt Certification
+# Test Cases
 
-Feature files for Firebolt Certification Modules.
+This directory contains the test cases that are used during the execution of the Firebolt Certification Suite (FCS).
```

Comment:
Can you emphasize that it is empty in the repo because it is cleared and copied over at the beginning of each run.

Reply:
It looks like you do explain for the most part but we should put this info at the top as well because people familiar with cypress will be confused.